app/assets/javascripts/views/project_view.js

```diff
@@ -109,6 +109,7 @@ module.exports = Backbone.View.extend({
 
     _.each(this.columns, function(column) {
       if(column.$el.hasClass('search_results_column')) return;
+      if(column.$el.hasClass('epic_column')) return;
```

Comment:
Why instead of adding another if you just don't add an `|| (or) `?

Reply:
Thought about that, but in my opinion this reads better than a multi line `if` with `||` or a long line using the two. 
What do you think?